iati/core/resources.py

```diff
@@ -109,12 +109,60 @@ def find_all_schema_paths(version=None):
     Todo:
         Handle versions, including errors.
 
-        Implement for more than a single specified activity schema.
+        Potentially add the IATI codelist schema.
+
+    """
+    return get_all_activity_schema_paths(version) + get_all_organisation_schema_paths(version)
+
+
+def get_all_activity_schema_paths(version=None):
+    """Find the paths for all activity schemas.
+
+    Args:
+        version (str): The version of the Standard to return the activity schemas for. Defaults to None. This means that paths to the latest version of the activity Schemas are returned.
+
+    Raises:
+        ValueError: When a specified version is not a valid version of the IATI Standard.
+
+    Returns:
+        list: A list of paths to all of the activity Schemas at the specified version of the Standard.
```

Comment:
The type should be `list of [something]`.

(same goes for other similar cases)